crates/cli/src/lib.rs

```diff
@@ -4,23 +4,22 @@ mod config;
 pub(crate) mod detect;
 mod edit_distance;
 mod errors;
-mod start;
 mod subcommands;
 mod tasks;
 pub mod util;
+pub mod version;
 
 use std::process::ExitCode;
 
 use clap::{ArgMatches, Command};
 
 pub use config::Config;
-use spacetimedb_paths::SpacetimePaths;
+use spacetimedb_paths::{RootDir, SpacetimePaths};
 pub use subcommands::*;
 pub use tasks::build;
 
 pub fn get_subcommands() -> Vec<Command> {
     vec![
-        version::cli(),
```

Comment:
is this meant to be replaced with `version list`?

Reply:
Yes, and it's added just further down.

Reply:
Ah it looks like `--version` does this now?